tests/unit/tests_standard_lib/test_requirement_tests/test_test_generator.py

```diff
@@ -133,11 +133,27 @@ def test_extract_params():
             ["requirement.xml"],
             [True],
             ["syslog"],
-            {"event": ["event_1", "event_2"]},
-            [["model_1:dataset_1", "model_2:dataset_2"], ["model_3:dataset_3"]],
-            ["event_name_2"],
+            {"event": ["<34>Oct 11 22:14:15 machine1 pr1:event_1"]},
+            [["model_1:dataset_1", "model_2:dataset_2"]],
+            ["event_name_1"],
             [{"field1": "value1", "field2": "value2"}, {"field3": "value3"}],
-            [],
+            [
```

Comment:
this test does not run on docker. Tests part of test_addon.py does

Reply:
@nandinivij ,
Docker test added
@arys-splunk , FYI